sailthru/sailthru_client.py

```diff
@@ -139,17 +134,17 @@ def set_email(self, email, _vars=None, lists=None, templates=None, verified=0, o
         lists = lists or []
         templates = templates or []
         send_vars = send_vars or []
-        data = {}
-        data['email'] = email
-        data['vars'] = _vars.copy()
-        data['lists'] = lists
-        data['templates'] = templates
-        data['verified'] = int(verified)
+        data = {'email': email,
+                'vars':  _vars.copy(),
+                'lists': lists,
+                'templates': templates,
+                'verified': int(verified)}
         if optout is not None:
             data['optout'] = optout
         if send is not None:
             data['send'] = send
-        data['send_vars'] = send_vars
+        if send_vars:
+            data['send_vars'] = send_vars
```

Comment:
might cause issue, when other part of the system calling the "send_vars" field in the dict, since the "send_vars" field will not be created.